payments/payment-methods/paypal/src/main/java/cm/aptoide/pt/payment_method/paypal/presentation/PaypalWebViewActivity.kt

```diff
@@ -0,0 +1,110 @@
+package cm.aptoide.pt.payment_method.paypal.presentation
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.webkit.CookieManager
+import android.webkit.WebResourceRequest
+import android.webkit.WebView
+import androidx.activity.addCallback
+import androidx.activity.compose.setContent
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import cm.aptoide.pt.payment_method.paypal.BuildConfig
+import cm.aptoide.pt.payment_method.paypal.Constants
+import com.google.accompanist.web.AccompanistWebChromeClient
+import com.google.accompanist.web.AccompanistWebViewClient
+import com.google.accompanist.web.WebView
+import com.google.accompanist.web.rememberWebViewState
+
+internal class PaypalWebViewActivity : AppCompatActivity() {
+  companion object {
+    internal const val EXTRA_URL = "${BuildConfig.LIBRARY_PACKAGE_NAME}.EXTRA_URL"
+    internal const val EXTRA_TOKEN = "${BuildConfig.LIBRARY_PACKAGE_NAME}.EXTRA_TOKEN"
+    internal const val FAIL = 100
+  }
+
+  private val url by lazy { intent.getStringExtra(EXTRA_URL) ?: "" }
+  private val token by lazy { intent.getStringExtra(EXTRA_TOKEN) ?: "" }
```

Comment:
This value is alway an empty string that is not used outside of this class, looks like it can be removed